setup.py

```diff
@@ -23,7 +23,7 @@
     'pytz',
     # install requirements depending on python version
     # see https://www.python.org/dev/peps/pep-0508/#environment-markers
-    'backports.zoneinfo; python_version == "3.7" or python_version == "3.8"',
+    'backports.zoneinfo; python_version <= "3.8"',
```

Comment:
This change would allow Python 3.6 and earlier. Will that actually work? If it cannot be confirmed, then I would revert this change.

Reply:
Nope .. all because `python_requires=">=3.8"`

Reply:
Actually that part is buggy `backports.zoneinfo` dependency should be `'backports.zoneinfo; python_version <= "3.9"',`

Reply:
```
Python 3.9.18 (main, Oct 23 2023, 13:20:13) 
>>> import zoneinfo
```

I would say `backports.zoneinfo; python_version < "3.9"`

I wonder if that would fox this in the doctest.
```
Expected:
    ZoneInfo(key='Europe/Vienna')
Got:
    zoneinfo.ZoneInfo(key='Europe/Vienna')
```

Reply:
https://github.com/kloczek/icalendar/pull/1 resolves this.